src/olympia/landfill/management/commands/generate_ui_test_addons.py

```diff
@@ -0,0 +1,32 @@
+from django.core.cache import cache
+from django.core.management.base import BaseCommand
+from django.core.management import call_command
+from django.utils import translation
+
+from olympia.landfill.serializers import GenerateAddonsSerializer
+
+
+class Command(BaseCommand):
+    """
+    Generate addons used specifically for the Integration Tests.
+
+    This will generate 10 addons with the name Ui-Addon, 1 Addon named
+    Ui-Addon-Test, 1 Featured theme, 4 featured collections, and 6 themes that
+    will not be marked as featured.
+
+    Usage:
+
+        python manage.py generate_ui_test_addons
+
+    """
+
+    def handle(self, *args, **kwargs):
+        translation.activate('en-US')
+        serializer = GenerateAddonsSerializer()
+        serializer.create_addons()
+        serializer.create_addon()
+        serializer.create_theme()
+        serializer.create_collections()
+        serializer.create_themes()
```

Comment:
This looks like things are a bit duplicate here and there. You're already calling `create_addons` and then again call `create_addon` so either you should give those methods proper names if they're creating different kinds of data (e.g the first might be just n dummy add-ons while the second one adds a very full featured add-on).

Worth some cleanup.